decred/tests/unit/util/test_chains.py

```diff
@@ -0,0 +1,21 @@
+"""
+Copyright (c) 2020, the Decred developers
+See LICENSE for details
+"""
+
+import pytest
+
+from decred import DecredError
+from decred.util import chains
+
+
+def test_parseCoinType():
+    with pytest.raises(DecredError):
+        chains.parseCoinType("not_a_coin")
+
+    assert chains.parseCoinType("DCR") == 42
+    assert chains.parseCoinType(42) == 42
+    assert chains.parseCoinType(-1) == -1
```

Comment:
Is the pass-through behavior when coinType is an integer what we actually want?

Reply:
It was my intention, yes. But I can see a case for checking that the value is a known asset, if that's what you're getting at.

Reply:
It is, but it doesn't have to happen in this PR.